recipes/random-name/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "random-name" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/random_name-{{ version }}.tar.gz
+  sha256: 9905292851b2adceabccafd8998e59acbe3a88e59ac2bd81f8fd9225bd46843b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This package declares `scripts` in the setup.py, which would prevent it from being noarch. However, the scripts folder is empty and there aren't any scripts. See https://github.com/phobologic/random_name/blob/0.1.1/setup.py#L31

```suggestion
  number: 0
  noarch: python
```